src/topology/stone_cech.lean

```diff
@@ -262,6 +262,21 @@ ultrafilter_extend_extends f
 lemma continuous_stone_cech_extend : continuous (stone_cech_extend hf) :=
 continuous_quot_lift _ (continuous_ultrafilter_extend f)
 
+lemma unique_stone_cech_extend {g: stone_cech α → γ} (hg: continuous g)
+(hfg: g ∘ stone_cech_unit = (stone_cech_extend hf) ∘ stone_cech_unit):
+g = stone_cech_extend hf :=
```

Comment:
```suggestion
  (hfg: g ∘ stone_cech_unit = (stone_cech_extend hf) ∘ stone_cech_unit) :
  g = stone_cech_extend hf :=
```
Just some minor spacing issues

Reply:
Is there an autoformatter for lean btw?

Reply:
> Is there an autoformatter for lean btw?

I don't know! That would be useful indeed!